Applications/MaterialManager/README.md

```diff
@@ -0,0 +1,16 @@
+# HOMAG materialManager Client
+
+The HOMAG materialManager Client enables access to the materialManager application for integration into your own applications. The functions offered on the website are available for a programmatic integration.
```

Comment:
Boris has currently a PR open with docu about mA. Please have a look and probably align that the description is similar.

Also there is an example how to create a MaterialAssist client. This would be also good here.

Reply:
I made some adjustments to have this more similar to what boris made. We already had the client creation in the authentification readme. now it is also here and the authentification readme was removed in favour of a global one